File: cmd/command/root.go
package command

import (
	"github.com/spf13/cobra"
)

func init() {
	RootCmd.AddCommand(setCmd, getCmd)
}

var RootCmd = &cobra.Command{
	Use:   "mr",
	Short: "mirror tool for repo, language, software(docker, container), etc...",
}

File: src/command/root.rs
use super::set;
use clap::Parser;

#[derive(Debug, Parser)]
#[command(name = "mr")]
#[command(bin_name = "mr")]
#[command(about = "mirror tool for repo, language, software(docker, container), etc...", long_about = None)]
pub enum Root {
    #[command(subcommand)]
    Set(set::Set),
}
